mdx2/command_line/map.py

```
@@ -2,56 +2,60 @@
-        type=float,
-        nargs=6,
-        metavar=("H1", "H2", "K1", "K2", "L1", "L2"),
-        help="region to map",
-    )
-    parser.add_argument("--signal", default="intensity", help="column in hkl_table to map")
-    parser.add_argument("--outfile", default="map.nxs", help="name of the output NeXus file")
+    def __post_init__(self):
+        """Validate limits parameter"""
+        hmin, hmax, kmin, kmax, lmin, lmax = self.limits
+        if hmin > hmax:
+            raise ValueError(f"limits: hmin must be <= hmax, got hmin={hmin}, hmax={hmax}")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)